ts/src/woo.ts

```diff
@@ -1070,27 +1288,26 @@ export default class woo extends Exchange {
         //
         // Possible input functions:
         // * createOrder
-        // * cancelOrder
         // * fetchOrder
         // * fetchOrders
         // const isFromFetchOrder = ('order_tag' in order); TO_DO
-        const timestamp = this.safeTimestamp2 (order, 'timestamp', 'created_time');
-        const orderId = this.safeString (order, 'order_id');
-        const clientOrderId = this.safeString (order, 'client_order_id'); // Somehow, this always returns 0 for limit order
+        const timestamp = this.safeInteger2 (order, 'timestamp', 'created_time');
+        const orderId = this.safeStringN (order, [ 'order_id', 'orderId', 'algoOrderId' ]);
+        const clientOrderId = this.safeString2 (order, 'client_order_id', 'clientOrderId'); // Somehow, this always returns 0 for limit order
         const marketId = this.safeString (order, 'symbol');
         market = this.safeMarket (marketId, market);
         const symbol = market['symbol'];
         const price = this.safeString2 (order, 'order_price', 'price');
         const amount = this.safeString2 (order, 'order_quantity', 'quantity'); // This is base amount
         const cost = this.safeString2 (order, 'order_amount', 'amount'); // This is quote amount
         const orderType = this.safeStringLower2 (order, 'order_type', 'type');
-        const status = this.safeValue (order, 'status');
+        const status = this.safeValue2 (order, 'status', 'algoStatus');
         const side = this.safeStringLower (order, 'side');
-        const filled = this.safeValue (order, 'executed');
-        const average = this.safeString (order, 'average_executed_price');
+        const filled = this.safeValue2 (order, 'executed', 'totalExecutedQuantity');
```

Comment:
@sc0Vu let's avoid using `safeValue` when is not strictly required

Reply:
@carlosmiei what do you mean about this?

Reply:
@sc0Vu I mean that we should use `safeString` or `safeNumber` here